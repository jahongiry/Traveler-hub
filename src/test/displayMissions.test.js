import { render, screen } from '@testing-library/react';
// import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import DisplayMissions from '../components/displayMissions';

describe('MissionDisplay', () => {
  test('Builds the snapchot of the displayMission component safely', () => {
    const displayMission = renderer.create(<DisplayMissions />);
    expect(displayMission.toJSON()).toMatchSnapshot();
  });

  test('renders DisplayMission component', () => {
    render(<DisplayMissions />);

    screen.getByRole('component');
  });
});
