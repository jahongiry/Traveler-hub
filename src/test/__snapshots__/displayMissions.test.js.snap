// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Missions component snapshot testing 1`] = `
<div
  className="missions-table"
>
  <table>
    <thead>
      <tr>
        <th>
          Mission
        </th>
        <th
          className="discription"
        >
          Description
        </th>
        <th>
          Status
        </th>
        <th>
          Join
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
